files-csharp/FastestWayToReadATextFileInCsharp/FastestWayToReadATextFileInCsharp/WaysToReadATextFileInCsharp.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System.Text;
+
+namespace FastestWayToReadATextFileInCsharp
+{
+    public class WaysToReadATextFileInCsharp
+    {
+        private readonly string _sampleFilePath;
+
+        public WaysToReadATextFileInCsharp(string sampleFilePath)
+        {
+            _sampleFilePath = sampleFilePath;
+        }
+
+        public string UseFileReadAllLines()
+        {
+            var stringArray = File.ReadAllLines(_sampleFilePath);
+
+            return string.Join(Environment.NewLine, stringArray);
+        }
+
+        public string UseFileReadAllText() => File.ReadAllText(_sampleFilePath);
+        
+        public string UseFileReadLines()
+        {
+            var stringArray = File.ReadLines(_sampleFilePath);
```

Comment:
Since ReadLines is returning an IEnumerable, we should probably rename this variable, maybe `lines`